src/main/webapp/placeGuides/MapPlaceGuideDisplayer.js

```diff
@@ -6,12 +6,81 @@ class MapPlaceGuideDisplayer {
     this._placeGuidesOnMap = {};
   }
 
+  static lngDistance(lngWest, lngEast) {
+    if (lngWest < lngEast) {
+      return lngEast - lngWest;
+    } else {
+      // West points distance from IDL + east points distance to IDL;
+      // (180 - lngWest) + (180 + lngEast);
+      return 360 - lngWest + lngEast;
+    }
+  }
+
+  // This function scans all the placeGuides to find the minimum map area
+  // which contains all of them.
+  // To do so, it finds the minimum and maximum latitude coordinate of all
+  // placeGudies. These will be set as the southern and northern bound of the
+  // map area.
+  // For longitudes, it firs finds the maximum lane which doesn't contain any
+  // placeGuides(for this purpose, placeGuides are sorted based on their
```

Comment:
This comment doesn't seem to belong here?

Reply:
You're right. Seems like one of the formatters that I applied changed the order of the comments and functions. 
I'm not sure about what happened, but I changed it manually.

Reply:
After a quick check, it seems like it was caused by the formatted in IntelliJ. I will investigate this later.